### DISCUSSION TITLE
Uncover tools

### DISCUSSION THREAD

**Discussion by Torch121** [projectdiscovery/uncover#137] [Q&A] [answered]
Hi bro how are you ? 
Please solution 

I have problem in uncover tools 
![Screenshot_20230102-223552_Termux](https://user-images.githubusercontent.com/55146714/210627617-ee92edba-c894-4714-a687-0fb277694cd4.jpg)

[FTL] Program exiting: no query provided
[FTL] Could not run enumeration: no keys provided

**Comment by forgedhallpass** [comment, marked as answer]
As the error message suggests, you need to provide API keys for one or more of the supported providers.
See the [provider configuration](https://github.com/projectdiscovery/uncover#provider-configuration) section on the readme page.

**Reply by Torch121** [reply]
![Screenshot_20230105-193949_Termux](https://user-images.githubusercontent.com/55146714/210855921-42a9318b-96ca-4f87-8bb1-3f3ecf83a902.jpg)
??
